spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -59,6 +59,21 @@
         expect(verification_array[7]).to eq(Moab::StorageObjectValidator::NO_XML_FILES =>"Version: v0006 Missing all required metadata files")
       end
     end
+    context 'under manifest directory' do
+      druid = 'cc000cc0000'
+      druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/cc/000/cc/0000/cc000cc0000'
+      storage_obj = Moab::StorageObject.new(druid, druid_path)
+      storage_obj_validator = described_class.new(storage_obj)
+      verification_array = storage_obj_validator.validation_errors
+      it 'does not have signatureCatalog.xml' do
+        expect(verification_array[0]).to eq(Moab::StorageObjectValidator::NO_SIGNATURE_CATALOG => "Version: v0001 Missing signatureCatalog.xml")
+      end
+
+      it 'does not have manifestInventory.xml' do
+        expect(verification_array[1]).to eq(Moab::StorageObjectValidator::NO_MANIFEST_INVENTORY => "Version: v0002 Missing manifestInventory.xml")
```

Comment:
this is fine.  Just for future reference, you could write 

```ruby
expect(verification_array).to include(Moab::StorageObjectValidator::NO_MANIFEST_INVENTORY => "Version: v0002 Missing manifestInventory.xml")
```

Reply:
ah okay makes sense

Reply:
stating the obvious:  it makes sense to do it your way if you care about which position in the array has the match.

Reply:
Okay - i'll update my code